datalad/interface/utils.py

```diff
@@ -363,33 +364,39 @@ def eval_func(wrapped, instance, args, kwargs):
                 def _result_filter(res):
                     return result_filter(res, **allkwargs)
 
-        def _get_procedure_specs(param_key=None, cfg_key=None, ds=None):
+        def _get_procedure_specs(param_key=None, cfg_key=None, ds=None, proc_cfg=None):
```

Comment:
The additional parameter is fine by me, though I have a slight preference for just defining `proc_cfg` above `_get_procedure_specs`.

Reply:
that is how I did it first, but then it should be not a pure variable since you can't rebind it inside, but some kind of a container (I had list).  Just looks ugly etc, so decided to make it explicit